src/parser/helper/language.ts

```diff
@@ -12,11 +12,11 @@ import JavaScript from "tree-sitter-javascript";
 // @ts-expect-error module doesn't have types
 import Kotlin from "tree-sitter-kotlin";
 // @ts-expect-error module doesn't have types
-import { php } from "tree-sitter-php";
+import PHP from "tree-sitter-php";
```

Comment:
Please give it a better name. PHP is pretty close to php. Try something like phpTreesitterConfig. Anything that makes it more clear